index.js

```diff
@@ -1,9 +1,13 @@
 /* @flow */
 /* eslint import/no-default-export: off */
 
-import { setupSDK, type SetupComponent } from '@paypal/sdk-client/src';
+import { setupSDK, type SetupComponent } from "@paypal/sdk-client/src";
 
 // $FlowFixMe
-export default (namespace : string, verison : string, components : $ReadOnlyArray<SetupComponent<mixed>>) => {
-    return setupSDK(components);
+export default (
+  namespace: string,
+  verison: string,
+  components: $ReadOnlyArray<SetupComponent<mixed>>
+) => {
```

Comment:
i think you need to delete that `// $FlowFixMe` above and move it down here
```suggestion
// $FlowFixMe
) => {
```